pom.xml

```diff
@@ -232,7 +232,9 @@
 	</dependencies>
 
 	<properties>
-		<skipTests>true</skipTests>
+	  <skipTests>false</skipTests>
```

Comment:
If you change this to false then some test will fails. The reason behind the failing test is due to some of them need ctools server credentials to get attachments for email archive, as those attachment are provided as links rather than encoded attachment string in the feed.